### DISCUSSION TITLE
Documentation URL Failed

### DISCUSSION THREAD

**Discussion by ValkyrieMashiro** [PKU-Alignment/safety-gymnasium#104] [Q&A]
I'm sorry to find out that the documentation URL is not working. The [URL](https://www.safety-gymnasium.com/) no longer points to thedocumentation of safety-gymnasium. I am eagerly awaiting a fix for this problem, thanks

**Comment by vrn-sn** [comment]
+1, our team is having the same issue.

**Comment by muchvo** [comment]
Thank you for bringing this to our attention. In the meantime, please make use of the documentation available at https://safety-gymnasium.readthedocs.io/en/latest/.
